Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,1500 @@ label mas_wrs_twitch:
+            eventlabel="mas_wrs_photoshop",
+            category=['photoshop'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_photoshop:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Doing some image editing, I see."
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_photoshop')
+    return    
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_camera",
+            category=['camera'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_camera:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Say cheese! Ahaha!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_camera')
+    return       
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_netscape",
+            category=['welcometonetscape!'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_netscape:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Hmmm, an old guide to the Internet? Don't you already know how this all works, though?\nEhehe, just wondering!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_netscape')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_unity",
+            category=['unity'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_unity:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Are you designing your own game? Very cool!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_unity')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_gms2",
+            category=['gamemakerstudio2'],
```

Comment:
I don't like the idea of adding version numbers to this purely because it means this must be maintained as GameMaker is updated. If it gets a v3 then this needs to change. Try to be as specific as possible in a way that doesn't require us to keep track of other projects' developments.